packages/client/README.md

```diff
@@ -249,9 +264,7 @@ await client.workspaces.deleteWorkspace(workspace);
 
 ## Deno support
 
-Right now we are still not publishing the client on deno.land or have support for deno in the codegen.
-
-However you can already use it with your preferred node CDN with the following import in the auto-generated `xata.ts` file:
+Right now we are still not publishing the client on [deno.land](https://deno.land) or have support for deno in the codegen. However you can already use it with your preferred Node.js CDN with the following import in the auto-generated `xata.ts` file:
```

Comment:
Is it worth putting a link to deno.land if we are saying we don't have anything published there yet?

Reply:
Should we begin publishing to deno.land?

We need to:

Agree on a name for the deno package. I'm okay with `xata` (they don't have namespaces like npm).

Set-up publishing releases to GitHub automatically.

Configure the webhook.

Can take care of that if we all agree 😎

Reply:
SGTM 👍